application-provider/src/main/java/org/jboss/intersmash/applications/ApplicationProvider.java

```diff
@@ -161,6 +174,26 @@ public static Path wildflyDistributedSessionsInfinispanDeploymentPath() {
 		return file;
 	}
 
+	/**
+	 * Provides access to a WAR deployment containing the
+	 * {@code wildfly-amq-broker-ssl} application
+	 *
+	 * @return {@link Path} instance that identifies the WAR artifact containing the
+	 *         {@code wildfly-amq-broker-ssl} application.
+	 */
+	public static Path wildflyActiveMQArtemisDeploymentPath() {
```

Comment:
rename wildflyActiveMQArtemisDeploymentPath to wildflyActiveMQArtemisSslDeploymentPath?